.evergreen/config.yml

```diff
@@ -2235,8 +2231,13 @@ buildvariants:
     auth: "auth"
     ssl: "nossl"
     encryption: "*"
-  display_name: "Encryption ${platform} ${auth} ${ssl}"
-  tasks: *encryption-server-versions
+  display_name: "Encryption ${platform} ${auth} ${ssl} ${encryption}"
```

Comment:
For all these display_name's we can do this: `${encryption} ${platform} ${auth} ${ssl}`. It will use the `display_name` defined in the axis values above (`Encryption` and `Encryption with CSFLE`)